### ISSUE TITLE
Load incorrectly compiling

### ISSUE THREAD

**Issue by Mastermori** [Stevertus/mcscript#9]
When you add variables in the load file, it doesn't compile the mcscript/load function correctly and puts its content in the normal load function which then looks like this:
`function kitpvp:mcscript/load

say Loaded

scoreboard players set @s test 10
execute as @a run say test
# Extended from data/kitpvp/functions/load.mcscript
# to .//kitpvp/data/kitpvp/functions/load.mcfunction
kitpvp/data/kitpvp/functions/mcscript/load
# please do not touch this file!
# it is used by the compiler!
scoreboard objectives add test dummy`

and thereby it doesnt load any of these files.

**Comment by Stevertus** [comment]
Could you try it with the new version? It might be fixed there

**Comment by Mastermori** [comment]
Yes, it is fixed now, you can close this issue, but there still are some mcscript/load bugs.
